canonical-web-and-design/maas.io
# Update header logo

## Done

Updated the header logo

## QA

- Go to https://maas-io-549.demos.haus/
- Check that the header logo has an orange background similar to ubuntu.com
